### DISCUSSION TITLE
关于生命周期顺序

### DISCUSSION THREAD

**Discussion by luy710** [NervJS/taro#10346] [Q & A] [answered]
vue3中，app.ts里面，进入页面相的关生命周期（如 onshow），是否都在page同名生命周期函数执行前执行？退出页面相的关生命周期（如 onhide），是否都在page同名生命周期函数执行前执行？并且是同步的？

**Comment by Chen-jj** [comment, marked as answer]
执行顺序不一定，看小程序什么时候触发 onShow/onHide。不是同步。
